petrometer/petrometer.py

```diff
@@ -41,19 +41,51 @@
 
 class Petrometer:
     def __init__(self, args: list):
-        parser = argparse.ArgumentParser(prog='petrometer')
-        parser.add_argument("addresses", metavar='ADDRESSES', nargs='+', type=str,
-                            help="Ethereum addresses to get the gas usage of")
-        parser.add_argument("--etherscan-api-key", help="Etherscan API key", required=True, type=str)
-        parser.add_argument("-j", '--json', help="Generate result as JSON", dest='json', action='store_true')
-        parser.add_argument("-o", "--output", help="File to save the output to", required=False, type=str)
-        parser.add_argument("-i", "--incoming", help="Show incoming transaction gas usage, defaut outgoing",
-                            required=False, action='store_true')
+        parser = argparse.ArgumentParser(prog="petrometer")
+        parser.add_argument(
+            "addresses",
+            metavar="ADDRESSES",
+            nargs="+",
+            type=str,
+            help="Ethereum addresses to get the gas usage of",
+        )
+        parser.add_argument(
+            "--etherscan-api-key", help="Etherscan API key", required=True, type=str
+        )
+        parser.add_argument(
+            "--ethfiller-key", help="Graphite API key", required=True, type=str
```

Comment:
@OleksandrUA why is this flag called `--ethfiller-key` if you're feeding it `GRAPHITE_KEY`? Probably something like `--graphite-key` should be more accurate.
Also, I think this tool should be used also if there is no need to send data to Graphite, so probably `required=False` would make more sense.